### ISSUE TITLE
int and int64's arent parsed (like at all)

### ISSUE THREAD

**Issue by Lite5h4dow** [goauthentik/helm#288]
bumped in to this issue trying to deploy the helm chart using the Terraform Helm Chart Resource.
got the following error

```
Error: template: authentik/templates/worker/deployment.yaml:34:28: executing "authentik/templates/worker/deployment.yaml" at <include (print $.Template.BasePath "/secret.yaml") .>: error calling include: template: authentik/templates/secret.yaml:13:6: executing "authentik/templates/secret.yaml" at <include "authentik.env" (dict "root" . "values" .Values.authentik)>: error calling include: template: authentik/templates/_helpers.tpl:24:20: executing "authentik.env" at <include "authentik.env" (dict "root" $.root "values" (dict (printf "%s__%s" (upper $k) (upper $sk)) $sv))>: error calling include: template: authentik/templates/_helpers.tpl:31:29: executing "authentik.env" at <$v>: wrong type for value; expected string; got int64
```

Looks like The terraform resource defaults to ints, not floats. template parsing shat the bed

PR: https://github.com/goauthentik/helm/pull/287

**Comment by rissson** [comment]
Closed by #287

**Comment by rissson** [comment]
Also, cross-linking to https://github.com/goauthentik/helm/issues/272, which seems to be the same issue
